src/components/pages/Registration/InstructorFlow_Step2.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import '../../../styles/registration.less';
+import InstructorFormSchema from './InstructorFormSchema';
+import * as yup from 'yup';
+
+const initialValues = {
+  name: '',
+  email: '',
+  location: '',
+  phone: '',
+  education: '',
+  tech: '',
+  notes: '',
+};
+
+const initialErrors = {
+  name: '',
+  email: '',
+  location: '',
+  phone: '',
+  education: '',
+};
+
+const initialSaveDisabled = true;
+
+const InstrRegForm = () => {
+  const [formValues, setFormValues] = useState(initialValues);
+  const [formErrors, setFormErrors] = useState(initialErrors);
+  const [disabled, setDisabled] = useState(initialSaveDisabled);
+
+  const validate = (name, value) => {
+    yup
+      .reach(InstructorFormSchema, name)
+      .validate(value)
+      .then(() => setFormErrors({ ...formErrors, [name]: '' }))
+      .catch(err => setFormErrors({ ...formErrors, [name]: err.errors[0] }));
+  };
+
+  const inputChange = (name, value) => {
+    validate(name, value);
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    });
+  };
+
+  const formSave = () => {
+    // TODO
+  };
+
+  const onSubmit = evt => {
+    evt.preventDefault();
+    formSave();
+    setFormValues(initialValues);
+  };
+
+  const onChange = evt => {
+    const { name, value } = evt.target;
+    inputChange(name, value);
+  };
+
+  useEffect(() => {
+    InstructorFormSchema.isValid(formValues).then(valid => setDisabled(!valid));
+  }, [formValues]);
+
+  return (
+    <div>
+      <div className="reg-content-container">
+        <div className="reg-form-header">
+          <h1>Instructor Account Info</h1>
+          <p className="indicator">*indicates required field</p>
+        </div>
+
+        <form className="basic-form" onSubmit={onSubmit}>
+          <label>
```

Comment:
Don't forget labels to make the forms more accessible

Reply:
Yeah, accessibility is one thing I need to add, as well as making this good for all screen sizes! Thank you!